File: slam/steps/binToRinex.mjs
// Заглушка — в самом upstream slamcloude/worker/pipeline/tasks.py этот шаг
// уже no-op (`if step is PipelineStep.BIN_TO_RINEX: ... return StepOutcome()`,
// см. tasks.py:495-502) вне зависимости от наличия ROVER_PPKRAW_BIN/BASE_BIN
// — переносится как есть, это не пробел порта, а факт исходной системы.
import { runStep } from "../lib/stepContext.mjs";

await runStep(async () => {
  return {};
});

File: pyserver/apps/slam/pipeline/steps/bin_to_rinex.py
"""
Stub -- in the upstream slamcloude/worker/pipeline/tasks.py itself this
step is already a no-op (`if step is PipelineStep.BIN_TO_RINEX: ... return
StepOutcome()`) regardless of ROVER_PPKRAW_BIN/BASE_BIN presence -- ported
as-is, this isn't a gap in the port, it's a fact of the source system.
"""


def run(ctx: dict) -> dict:
    return {}

